cmd/kubeadm/app/util/config/initconfiguration_test.go

```diff
@@ -23,9 +23,9 @@ import (
 	"path/filepath"
 	"testing"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
in CAPI we are standardising this to `corev1`. Should we do the same here? (if not, I guess we can drop the alias)

Reply:
the alias is added automatically by goimports IIRC.
the same is present in a number of places in k8s and kubeadm: https://github.com/kubernetes/kubernetes/search?q=path%3A%2Fcmd%2Fkubeadm+%22k8s.io%2Fapi%2Fcore%2Fv1%22

Reply:
corev1 is used in a number of places too it seems.
https://github.com/kubernetes/kubernetes/search?q=corev1 

looks like there is no standardization.
it can be done in a follow up.